src/demos/cern-videos/App.js

```diff
@@ -48,8 +48,10 @@ const resultsPerPageValues = [
 ];
 
 const searchApi = new InvenioSearchApi({
-  url: 'https://videos.cern.ch/api/records/',
-  timeout: 5000,
+  axios: {
```

Comment:
@topless Thanks for fixing that I totally missed it :(

Reply:
@zzacharo no worries! I have fixed everything on ILS side too!